### PR TITLE
Implement URL alias view action override through semantic config

### PR DESCRIPTION
**Pull request by pspanja**

### CODE REVIEW

**Comment by pspanja** [comment]
Ping @emodric

File: bundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        $this->addConfiguration($rootNode);
+
+        return $treeBuilder;
+    }
+
+    protected function addConfiguration($rootNode)
+    {
+        $systemNode = $this->generateScopeBaseNode($rootNode);
+        $systemNode
+            ->booleanNode('override_url_alias_view_action')
```
Comment (emodric) [inline bundle/DependencyInjection/Configuration.php:31]: How about just `override_default_controller` ?

**Comment by emodric** [comment]
Other than the inline comment, +1.

File: bundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (pspanja) [inline bundle/DependencyInjection/Configuration.php:31]: The thing is there is no such concept in the code, this actually **is** the action when the view is resolved through URL alias.

**Comment by pspanja** [comment]
Thanks!

**Pull request merged**